style.css

```diff
@@ -1,1990 +1,4132 @@
+    object-fit: cover;
+
+    /* Neutrals/N0 */
+    border: 1px solid #fff;
+  }
+
+  .shape14 {
+    position: absolute;
+    width: 48px;
+    height: 48px;
+    left: 285px;
+    top: 150px;
+    background: url(Images/Ellipse2.png);
+    object-fit: cover;
+
+    /* Neutrals/N0 */
+    border: 1px solid #fff;
+  }
+
+  .shape15 {
+    position: absolute;
+    width: 48px;
+    height: 48px;
+    left: 420px;
+    top: 150px;
+    background: url(Images/Ellipse3.png);
+
+    /* Neutrals/N0 */
+    border: 1px solid #fff;
+  }
+
+  .FRAME {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    width: 327px;
+    height: 320.15px;
+    position: relative;
+  }
+
+  .FRAME a {
+    /* Languages */
+    position: absolute;
+    height: 24px;
+    left: -60px;
+    right: 40px;
+    top: -50px;
+
+    /* Body / 1 - Bolder */
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    text-decoration: none;
+
+    /* identical to box height, or 120% */
+    display: flex;
+    align-items: center;
+    color: #344563;
+  }
+
+  .SKILL {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    width: 327px;
+    height: 320.15px;
+  }
+
+  .SKILL a {
+    /* Languages */
+    position: absolute;
+    height: 24px;
+    left: 140px;
+    right: 20px;
+    top: 350px;
+
+    /* Body / 1 - Bolder */
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    text-decoration: none;
+
+    /* identical to box height, or 120% */
+    display: flex;
+    align-items: center;
+    color: #344563;
+  }
+
+  .arr1 {
+    position: absolute;
+    height: 8px;
+    right: 30px;
+    top: 87px;
+    width: 14px;
+    list-style: none;
+  }
+
+  .arr2 {
+    position: absolute;
+    height: 14px;
+    right: 30px;
+    bottom: 60px;
+    width: 8px;
+    list-style: none;
+  }
+
+  .arr3 {
+    position: absolute;
+    height: 14px;
+    right: 30px;
+    bottom: 1px;
+    width: 8px;
+    list-style: none;
+  }
+
+  .arr1:hover {
+    background: none;
+  }
+
+  .footer::before {
+    content: "";
+    background-image: url(Images/contactformbackgroundshapes.png);
+    background-position: right;
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 79%;
+    position: absolute;
+    top: 80px;
+    left: -70px;
+    right: 74px;
+    border-top-left-radius: 5rem;
+    z-index: -1;
+  }
+
+  .footer {
+    position: relative;
+    top: -145px;
+    width: 1440px;
+    height: 846px;
+    left: 1px;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    z-index: 10;
+    background: url(Images/contactformbackgroundshapedesktop.png);
+    background-position: 50% 50%;
+    background-size: cover;
+  }
```

Comment:
- [ ] I noticed a space below your footer, Kindly remove the space so your portfolio will match the design template.
- [ ] Please center your contact form to match the design template.
## Your design
![d3](https://user-images.githubusercontent.com/94551484/186765155-6835ce10-b288-4b2c-869b-95dfd9666df7.png)
## Figma template
![zzz](https://user-images.githubusercontent.com/94551484/186765196-4dadf207-0fe7-49b2-9e50-aa6f490dbe0e.png)

- [Optional] Consider removing all static heights and widths. It is best in responsive design not to use static heights and widths. You can use `height: auto;` and `width` in percentage `%` instead.